### ISSUE TITLE
Training on SCARF Embeddings results in worse accuracy

### ISSUE THREAD

**Issue by golnaz-amii** [clabrugere/pytorch-scarf#5]
Hi!

I'm new to SSL for tabular data, and was wondering why in `example/example.ipynb` model performance is worse on the embedding than raw features, and if there are ways to debug/finetune/tune the hypers so that SCARF actually helps

Thank you
